source/sourcing_and_processing_data/processors/simplify_geo_shape.rst

```diff
@@ -0,0 +1,19 @@
+Simplify Geo Shape processor
+============================
+
+This processor simplifies a geo shape to reduce both the processing time and the dataset size.
+
```

Comment:
We can add a note about the fact that the simplify process won't respect the topology of the shape (sorry)